dockerfile/cuda11.8.dockerfile

```diff
@@ -9,8 +9,8 @@ FROM nvcr.io/nvidia/pytorch:22.12-py3
 #   - cuDNN: 8.7.0.84
 #   - NCCL: v2.15.5-1
 # Mellanox:
-#   - OFED: 5.2-2.2.3.0
-#   - HPC-X: v2.8.3
+#   - OFED: 5.8-1.0.1.1
+#   - HPC-X: v2.13.1
```

Comment:
@abuccts please confirm it.

Reply:
Also, do we want separated PR to upgrade OFED and HPC-X? It would be easy to track.

Reply:
reverted